http_examples.cpp

```diff
@@ -57,13 +57,18 @@ int main() {
 
             string name=pt.get<string>("firstName")+" "+pt.get<string>("lastName");
 
-            *response << "HTTP/1.1 200 OK\r\nContent-Length: " << name.length() << "\r\n\r\n" << name;
+            *response << "HTTP/1.1 200 OK" << "\r\n"
+                      << "Access-Control-Allow-Origin: *" << "\r\n" 
```

Comment:
I tested this on newest Firefox, and I do not think `Access-Control-Allow-Origin` is needed?